dagger-functions/src/main/java/io/odpf/dagger/functions/udfs/python/PythonUdfConfigMapper.java

```diff
@@ -0,0 +1,96 @@
+        pythonKeyMap.put(PYTHON_REQUIREMENTS_KEY, "python.requirements");
+        pythonKeyMap.put(PYTHON_ARCHIVES_KEY, "python.archives");
+        pythonKeyMap.put(PYTHON_CLIENT_EXECUTABLE_KEY, "python.client.executable");
+        pythonKeyMap.put(PYTHON_EXECUTABLE_KEY, "python.executable");
+        pythonKeyMap.put(PYTHON_FN_EXECUTION_ARROW_BATCH_SIZE_KEY, "python.fn-execution.arrow.batch.size");
+        pythonKeyMap.put(PYTHON_FN_EXECUTION_BUNDLE_SIZE_KEY, "python.fn-execution.bundle.size");
+        pythonKeyMap.put(PYTHON_FN_EXECUTION_BUNDLE_TIME_KEY, "python.fn-execution.bundle.time");
+        pythonKeyMap.put(PYTHON_FN_EXECUTION_MEMORY_MANAGED_KEY, "python.fn-execution.memory.managed");
+        pythonKeyMap.put(PYTHON_MAP_STATE_ITERATE_RESPONSE_BATCH_SIZE_KEY, "python.map-state.iterate-response-batch-size");
+        pythonKeyMap.put(PYTHON_MAP_STATE_READ_CACHE_SIZE_KEY, "python.map-state.read-cache-size");
+        pythonKeyMap.put(PYTHON_MAP_STATE_WRITE_CACHE_SIZE_KEY, "python.map-state.write-cache-size");
+        pythonKeyMap.put(PYTHON_STATE_CACHE_SIZE_KEY, "python.state.cache-size");
+        pythonKeyMap.put(PYTHON_METRIC_ENABLED_KEY, "python.metric.enabled");
+        pythonKeyMap.put(PYTHON_PROFILE_ENABLED_KEY, "python.profile.enabled");
+        return Collections.unmodifiableMap(pythonKeyMap);
+    }
+
+    private static Map<String, Class> createTypeMap() {
+        Map<String, Class> pythonTypeMap = new HashMap<>();
+        pythonTypeMap.put(PYTHON_FILES_KEY, String.class);
+        pythonTypeMap.put(PYTHON_REQUIREMENTS_KEY, String.class);
+        pythonTypeMap.put(PYTHON_ARCHIVES_KEY, String.class);
+        pythonTypeMap.put(PYTHON_CLIENT_EXECUTABLE_KEY, String.class);
+        pythonTypeMap.put(PYTHON_EXECUTABLE_KEY, String.class);
+        pythonTypeMap.put(PYTHON_FN_EXECUTION_ARROW_BATCH_SIZE_KEY, Integer.class);
+        pythonTypeMap.put(PYTHON_FN_EXECUTION_BUNDLE_SIZE_KEY, Integer.class);
+        pythonTypeMap.put(PYTHON_FN_EXECUTION_BUNDLE_TIME_KEY, Long.class);
+        pythonTypeMap.put(PYTHON_FN_EXECUTION_MEMORY_MANAGED_KEY, Boolean.class);
+        pythonTypeMap.put(PYTHON_MAP_STATE_ITERATE_RESPONSE_BATCH_SIZE_KEY, Integer.class);
+        pythonTypeMap.put(PYTHON_MAP_STATE_READ_CACHE_SIZE_KEY, Integer.class);
+        pythonTypeMap.put(PYTHON_MAP_STATE_WRITE_CACHE_SIZE_KEY, Integer.class);
+        pythonTypeMap.put(PYTHON_STATE_CACHE_SIZE_KEY, Integer.class);
+        pythonTypeMap.put(PYTHON_METRIC_ENABLED_KEY, Boolean.class);
+        pythonTypeMap.put(PYTHON_PROFILE_ENABLED_KEY, Boolean.class);
+
+        return Collections.unmodifiableMap(pythonTypeMap);
+    }
+
+    public Map<String, Object> getConfig() {
+        Map<String, String> configMap = pythonUdfConfig.jsonToConfigMap();
+
+        Map<String, Object> config = new HashMap<>();
+        for (Map.Entry<String, String> entry : configMap.entrySet()) {
+            if (getType(entry.getKey()) == Long.class) {
```

Comment:
GSON can handle the types too right? In the configuration we can have correct datatype when creating this POJO? is it not possible? 
```
    @SerializedName(PYTHON_FN_EXECUTION_BUNDLE_TIME_KEY)
    @Getter
    private long pythonBundleTime;
```